src/main/groovy/com/ullink/NuGetExtension.groovy

```diff
@@ -2,4 +2,5 @@ package com.ullink
 
 class NuGetExtension {
     String version = '3.4.3'
+    String nugetUrl = null
```

Comment:
I think the default for nuget version should be null (aka 'latest'). Otherwise you are forced to constantly modify plugin in order to keep up with up-to-date nuget version.

//  src/main/groovy/com/ullink/NuGetExtension.groovy 
class NuGetExtension {
     String version = null
     String nugetUrl = null

// src/main/groovy/com/ullink/BaseNuGet.groovy 
version = project.extensions.nuget.version
nugetUrl = project.extensions.nuget.nugetUrl
    ?: "https://dist.nuget.org/win-x86-commandline/" +  (version ? 'v' + version : 'latest') + '/nuget.exe'